components/org.wso2.carbon.identity.api.user.application/org.wso2.carbon.identity.rest.api.user.application.v1/.openapi-generator-ignore

```diff
@@ -0,0 +1,26 @@
+# OpenAPI Generator Ignore
```

Comment:
I think this file is not needed.

Reply:
This file is something similar to gitignore.
Specifically committed as we should avoid regenerating the service impl at /impl path

Reply:
Yes. But the added file only has commented items :)

Reply:
Not the last line. I will update the file such that only necessary content is there

Reply:
Fixed